contrib/labstack/echo/echotrace.go

```diff
@@ -0,0 +1,48 @@
+// Package echo provides functions to trace the labstack/echo package (https://github.com/labstack/echo).
+package echo
+
+import (
+	"strconv"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+
+	"github.com/labstack/echo"
+)
+
+// Middleware returns middleware that will trace incoming requests.
+func Middleware(service string) echo.MiddlewareFunc {
```

Comment:
I think we'll need to look like this:
```go
func Middleware(opts ...MiddlewareOption) echo.MiddlewareFunc {
```
You can use our `gorilla/mux` integration as inspiration (see [here](https://github.com/DataDog/dd-trace-go/blob/61d8c99c511e0edf8b27364bd6cc053623720662/contrib/gorilla/mux/mux.go#L69) and [here](https://github.com/DataDog/dd-trace-go/blob/61d8c99c511e0edf8b27364bd6cc053623720662/contrib/gorilla/mux/option.go)).

The reason is that if we need to add more options it will not be a breaking change for older users. Right now, if we were to need a new argument, it would be a breaking change.